spaces/S000105/properties/P000199.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000105
+property: P000199
+value: true
+---
+
+Let $g$ be a function in Helly space. Since Helly space is closed under convex combination, $H(f, t) := (1-t)f + tg$ defines a homotopy $H : X \times [0, 1] \to X$ from the identity to a constant map.
```

Comment:
```suggestion
Let $\iota:X\to X$ be the identity and $z:X\to X$ be the constant map to the zero function $0\in X$. Then $H(f, t) := (1-t)f$ defines a continuous function $H : X \times [0, 1] \to X$ satisfying $H(f,0)=\iota(f)$ and $H(f,1)=z(f)$, showing the identity $\iota$ and constant map $z$ are [homotopic](https://en.wikipedia.org/wiki/Homotopy#Formal_definition).
```

Maybe this more directly fits the definition we have in P199?

Reply:
Sounds good to me. Also I didn't notice it's a lot better to use $g = 0$.